definitions/npm/react-redux_v5.x.x/flow_v0.89.x-/react-redux_v5.x.x.js

```diff
@@ -79,7 +79,7 @@ declare module "react-redux" {
   // Got error like inexact OwnProps is incompatible with exact object type?
   // Just make the OP parameter for `connect()` an exact object.
   declare type MergeOP<OP, D> = {| ...$Exact<OP>, dispatch: D |};
-  declare type MergeOPSP<OP, SP> = {| ...$Exact<OP>, ...SP |};
+  declare type MergeOPSP<OP, SP, D> = {| ...$Exact<OP>, ...SP, dispatch: D |};
```

Comment:
👍

Reply:
@AndrewSouthpaw I just spent a while tracking this down as it was tripping me up for a while and I couldn't understand why I should be expected to add a dispatch property to my exact Props type, a helpful comment above this line wouldn't go amiss. Thanks for your work by the way, highly appreciated! 👍